### PR TITLE
[WalletApp] Import account and settings screen

### PR DESCRIPTION
**Pull request by flypaper0**
# Description

Resolves # (issue)

## Due Dilligence

* [ ] Breaking change
* [ ] Requires a documentation update

### CODE REVIEW

File: Example/WalletApp/PresentationLayer/Wallet/Settings/SettingsView.swift [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                        .frame(maxWidth: .infinity)
+                }
+            }
+        }
+        .listStyle(.insetGrouped)
+        .alert("Value copied to clipboard", isPresented: $copyAlert) {
+            Button("OK", role: .cancel) { }
+        }
+        .onAppear {
+            viewModel.objectWillChange.send()
```
Comment (radeknovis) [inline Example/WalletApp/PresentationLayer/Wallet/Settings/SettingsView.swift:36]: Why this?

File: Example/WalletApp/PresentationLayer/Wallet/Settings/SettingsView.swift [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+                Text(subtitle)
+                    .multilineTextAlignment(.trailing)
+                    .foregroundColor(.gray)
+            }
+        }
+    }
+}
+
+#if DEBUG
```
Comment (radeknovis) [inline Example/WalletApp/PresentationLayer/Wallet/Settings/SettingsView.swift:60]: nit: Xcode does this automatically for PreviewProvider struct so no need to do explicitly

File: Example/WalletApp/PresentationLayer/Wallet/Welcome/Views/TextFieldView.swift [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import SwiftUI
+
+struct TextFieldView: View {
+
+    let title: String
+    let placeholder: String
+    let input: Binding<String>
```
Comment (radeknovis) [inline Example/WalletApp/PresentationLayer/Wallet/Welcome/Views/TextFieldView.swift:7]: nit: you don't fancy @Binding property wrapper? :P

**Review by radeknovis — APPROVED** [review]

File: Example/WalletApp/PresentationLayer/Wallet/Settings/SettingsView.swift (hunk 1 above)
Comment (flypaper0) [reply Example/WalletApp/PresentationLayer/Wallet/Settings/SettingsView.swift:36]: To get actual data on every on appear

**Review by llbartekll — APPROVED** [review]

File: Sources/Web3Inbox/PushClientProxy/PushClientProxy.swift [hunk 4]
```diff
@@ -7,9 +7,13 @@ final class PushClientProxy {
     var onSign: SigningCallback
     var onResponse: ((RPCResponse) async throws -> Void)?
 
-    init(client: WalletPushClient, onSign: @escaping SigningCallback) {
+    init(account: Account, client: WalletPushClient, onSign: @escaping SigningCallback) {
         self.client = client
         self.onSign = onSign
+
+        Task {
```
Comment (llbartekll) [inline Sources/Web3Inbox/PushClientProxy/PushClientProxy.swift:14]: won't be any issues with .priority ?

**Review by llbartekll — APPROVED** [review]

**Pull request merged**